server/controllers/appointmentController.js

```diff
@@ -10,182 +10,190 @@ const twilioNumber = process.env.TWILIO_NUMBER;
 // user id should be passed in through :id params
 // must pass in a stylist and date in format "2018-08-22T12:12:12.764Z"
 // Will hook up user's number when closer to production build
 const createAppointment = (req, res) => {
-	const user = req.params.id;
-	const { stylist, session, service } = req.body;
-	console.log(req.body);
-	const appointment = new Appointment({
-		user,
-		stylist: stylist._id,
-		session,
-		service
-	});
-	appointment
-		.save()
-		.then(appt => {
-			// This trims the date and time to send to the user through text
-			let apptDay = session.slice(5, 7) + "/" + session.slice(8, 10);
-			let apptTime = session.slice(11, 16);
-			if (Number(apptTime.slice(0, 2)) > 12)
-				apptTime =
-					apptTime.replace(
-						apptTime.slice(0, 2),
-						Number(apptTime.slice(0, 2)) - 12
-					) + " PM";
-			else apptTime += " AM";
+  const user = req.params.id;
+  const { stylist, session, service } = req.body;
+  console.log(req.body);
+  const appointment = new Appointment({
+    user,
+    stylist: stylist._id,
+    session,
+    service
+  });
+  appointment
+    .save()
+    .then(appt => {
+      // This trims the date and time to send to the user through text
+      let apptDay = session.slice(5, 7) + "/" + session.slice(8, 10);
+      let apptTime = session.slice(11, 16);
+      if (Number(apptTime.slice(0, 2)) > 12)
+        apptTime =
+          apptTime.replace(
+            apptTime.slice(0, 2),
+            Number(apptTime.slice(0, 2)) - 12
+          ) + " PM";
+      else apptTime += " AM";
 
-			// Twilio integration
-			// send text here
-			// client.messages
-			// 	.create({
-			// 		body: `Your appointment with ${
-			// 			stylist.name
-			// 		} on ${apptDay} at ${apptTime} has been scheduled!`,
-			// 		to: myNumber, // Number to send text to. Put your number here to test
-			// 		from: twilioNumber // From a valid Twilio number
-			// 	})
-			// 	.then(message => console.log(message.sid))
-			// 	.catch(err => console.log(err));
+      // Twilio integration
+      // send text here
+      // client.messages
+      // 	.create({
+      // 		body: `Your appointment with ${
+      // 			stylist.name
+      // 		} on ${apptDay} at ${apptTime} has been scheduled!`,
+      // 		to: myNumber, // Number to send text to. Put your number here to test
+      // 		from: twilioNumber // From a valid Twilio number
+      // 	})
+      // 	.then(message => console.log(message.sid))
+      // 	.catch(err => console.log(err));
 
-			res.status(200).json({
-				success: "Appointment saved",
-				appt
-			});
-		})
-		.catch(err => {
-			console.log(err);
-			res.status(400).send({ error: err });
-		});
+      res.status(200).json({
+        success: "Appointment saved",
+        appt
+      });
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(400).send({ error: err });
+    });
 };
 
+let twilioReminder = new CronJob("0 18 * * * * *", function() {
+  client.messages.create({
```

Comment:
Once we get a full Twilio account, we can change this to have take an array of User's phone numbers